liquidator/test/Liquidator.js

```diff
@@ -163,7 +163,8 @@ contract("Liquidator.js", function(accounts) {
     // Start with a mocked price of 1 usd per token.
     // This puts both sponsors over collateralized so no liquidations should occur.
     priceFeedMock.setCurrentPrice(toBN(toWei("1")));
-    await liquidator.queryAndLiquidate();
+    await liquidator.update();
```

Comment:
tests only changed to accommodate the new function signature and to explicitly call `update` before hand.